plugins/shell-proxy/ssh-proxy.py

```diff
@@ -21,8 +21,12 @@
     raise TypeError('unsupported proxy protocol: "{}"'.format(parsed.scheme))
 
 def make_argv():
-    yield "nc"
-    if sys.platform in {'linux', 'cygwin'}:
+    if sys.platform == 'darwin':
+        # 'nc' in $PATH may be installed by homebrew, if without path
+        yield "/usr/bin/nc"
+    else:
+        yield "nc"
```

Comment:
hardcoded path is bad code.

i think it is needed by environ.

```suggestion
    yield os.environ.get("SHELLPROXY_NETCAT_PROGRAM", "nc")
```

Reply:
/usr/bin/nc is the BSD version pre-installed in MacOS，If users need to configure the system using SHELLPROXY_NETCAT_PROGRAM, then it might require debugging to identify the issue.